h/static/styles/variables.scss

```diff
@@ -42,6 +42,21 @@ $success-color: #1cbd41 !default;
 $mask-start-color: rgba($white, 0) !default;
 $mask-end-color: $white !default;
 
+
+// Variables for the new color palette
+// ------------------------------------
+$grey-1: #F2F2F2;
```

Comment:
There is another set of 'Variables for the new color palette' at the bottom of this file, with names like `lg-1`, `dg-2`. Can you make sure that we only have one set (eg. remove the others and replace with references to these ones)